cmd/drone-server/server.go

```diff
@@ -157,28 +158,32 @@ var flags = []cli.Flag{
 	// resource limit parameters
 	//
 	cli.Int64Flag{
-		Name:   "limit-mem-swap",
 		EnvVar: "DRONE_LIMIT_MEM_SWAP",
+		Name:   "limit-mem-swap",
+		Usage:  "maximum swappable memory allowed in bytes",
 	},
 	cli.Int64Flag{
-		Name:   "limit-mem",
 		EnvVar: "DRONE_LIMIT_MEM",
+		Name:   "limit-mem",
+		Usage:  "maximum memory allowed in bytes",
 	},
 	cli.Int64Flag{
-		Name:   "limit-shm-size",
 		EnvVar: "DRONE_LIMIT_SHM_SIZE",
+		Name:   "limit-shm-size",
 	},
 	cli.Int64Flag{
-		Name:   "limit-cpu-quota",
 		EnvVar: "DRONE_LIMIT_CPU_QUOTA",
+		Name:   "limit-cpu-quota",
+		Usage:  "impose a cpu quota",
 	},
 	cli.Int64Flag{
-		Name:   "limit-cpu-shares",
 		EnvVar: "DRONE_LIMIT_CPU_SHARES",
+		Name:   "limit-cpu-shares",
+		Usage:  "change the cpu shares",
 	},
 	cli.StringFlag{
-		Name:   "limit-cpu-set",
 		EnvVar: "DRONE_LIMIT_CPU_SET",
+		Name:   "limit-cpu-set",
```

Comment:
No usage here, couldn't find any docs about this on https://docs.docker.com/engine/admin/resource_constraints/.

Reply:
https://docs.docker.com/compose/compose-file/compose-file-v2/#cpu-and-other-resources

Reply:
That got me to https://docs.docker.com/engine/reference/run/#cpuset-constraint.

Does this refer to `--cpuset-cpus` or `--cpuset-mems`?

Reply:
I'm not sure about this one, need to dig into the code

Reply:
Is cpu set for [CPUs](https://github.com/cncd/pipeline/blob/82706703c87f88a632674484e095e494c0e30e34/pipeline/backend/docker/convert.go#L43)

Reply:
Thanks

Reply:
Thanks, added.